tests/integration/frameworks/models/catboost.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+import typing as t
+
+from sklearn import metrics
+from catboost import CatBoostClassifier
+from sklearn.datasets import make_classification
+from sklearn.model_selection import train_test_split
+
+import bentoml
+
+from . import FrameworkTestModel
+from . import FrameworkTestModelInput as Input
+from . import FrameworkTestModelConfiguration as Config
+
+framework = bentoml.catboost
+
+
+def accurate_to(expected, accuracy: float) -> t.Callable[..., bool]:
```

Comment:
just a quick type suggestion

```suggestion
import bentoml._internal.external_typing as ext

def accurate_to(expected: ext.NpNdArray, accuracy: float) -> t.Callable[[ext.NpNdArray], bool]:
```